cardano-cli/src/Cardano/CLI/EraBased/Common/Option.hs

```diff
@@ -2,6 +2,7 @@
 {-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE OverloadedRecordDot #-}
```

Comment:
I think this will break compilation on 8.10 in cardano-node since this is since 9.2.

Reply:
I have updated the PR to not use the extension.  Can squash if this is acceptable.